manifests/init.pp

```diff
@@ -205,6 +205,7 @@
   $service_restart                                           = undef,
   $service_name                                              = 'nginx',
   $service_manage                                            = true,
+  Boolean $service_config_check                              = false,
```

Comment:
Please add puppet-strings documentation for this new parameter :)

Reply:
Done.